examples/CMakeLists.txt

```
@@ -447,12 +447,30 @@ add_impactx_test(solenoid.MADX.py
     OFF  # no plot script yet
 )
 
+# Pole-face rotation ###################################################
+#
+# w/o space charge
+add_impactx_test(rotation
+    examples/rotation/input_rotation.in
+      ON  # ImpactX MPI-parallel
```

Just some alignment:
```suggestion
      ON   # ImpactX MPI-parallel
```